docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -299,12 +299,70 @@ folder in the project root during development.
 
 ## Running in Production
 
+Cube Store is enabled by default when running Cube.js in development mode. In
+production, Cube Store **must** run as a separate process. The easiest way to do
+this is to use the official Docker images for Cube.js and Cube Store.
+
+You can run Cube Store with Docker with the following command:
+
+```bash
+docker run -p 3030:3030 cubejs/cubestore
+```
+
+<!-- prettier-ignore-start -->
+[[info | ]]
+| Cube Store can further be configured via environment variables. To see a
+| complete reference, please consult the [Cube Store section of the Environment
+| Variables reference][ref-config-env].
+<!-- prettier-ignore-end -->
+
+Next, run Cube.js and tell it to connect to Cube Store running on `localhost` on
+port `3030`:
+
+```bash
+docker run -p 4000:4000 -e CUBEJS_CUBESTORE_HOST=localhost -e CUBEJS_CUBESTORE_PORT=3030 -v ~/cube-conf:/cube/conf/ cubejs/cube
```

Comment:
@hassankhan is it required to specify the `CUBEJS_CUBESTORE_PORT`? I'm leaning towards to keep as mich clear as possible and then link to the reference for additional options and overrides.